lib/ansible/modules/network/iosxr/iosxr_facts.py

```diff
@@ -54,7 +54,8 @@
         can also be used with an initial C(M(!)) to specify that a
         specific subset should not be collected.
     required: false
-    choices: ['all', 'lacp', '!lacp', 'lacp_interfaces', '!lacp_interfaces', 'interfaces', '!interfaces']
+    choices: ['all', 'lacp', '!lacp', 'lacp_interfaces', '!lacp_interfaces', 'interfaces', '!interfaces',
```

Comment:
could you also include lldp_global and lldp_interfaces and their negated versions here?

Reply:
done